psydac/ddm/petsc.py

```diff
@@ -0,0 +1,102 @@
+# coding: utf-8
+
+import numpy as np
+from itertools import product
+
+from .cart import CartDecomposition
+
+#===============================================================================
+class PetscCart:
+
+    def __init__(self, cart):
+        assert isinstance(cart, CartDecomposition)
+
+        try:
+            from petsc4py import PETSc
+        except ImportError:
+            raise 
```

Comment:
Please add a docstring to this new class.

And can we raise a more intelligible error, where we say clearly that `petsc4py` must be installed with `pip` in order to use class `PetscCart`?

Reply:
it's more difficult than that, for it to work  the user needs to install petsc first and then petsc4py , I think the import error should be enough

Reply:
But why are you using a try/except clause, if not for giving a nice error for the user??

Reply:
If you use a `try/except` block, you should give a proper error. Otherwise you can just remove the `try/except` block...

Reply:
yes that's true

Reply:
done